rabbit-escape-engine/test/rabbitescape/engine/TestTextWorldManip.java

```diff
@@ -457,6 +458,21 @@ public void Can_provide_empty_description()
         );
     }
 
+    @Test
+    public void Can_obfuscate_hints()
+    {
+        String[] lines = {
+            ":hint1.code=_Uf?>3ZH_8>U>{3",
+            ":hint2.code=@XW@W:)e+:+ ",
+            ":hint3.code=X5g..T[6X4["
+        };
+
+        World world = createWorld( lines );
+
+        assertEquals( world.hint1, "Select the bash" );
```

Comment:
@colonelfazackerley junit.Assert.assertEquals also gives useful messages on failures. The main reason for using it was that I have junit.Assert set up with autosuggest in Eclipse.